src/valkey-cli.c

```diff
@@ -7310,6 +7621,48 @@ static int clusterManagerCommandFix(int argc, char **argv) {
     return clusterManagerCommandCheck(argc, argv);
 }
 
+static int clusterApplyReshardTable(list *table, clusterManagerNode *target, int opts) {
+    listIter li;
+    listNode *ln;
+    listRewind(table, &li);
+    while ((ln = listNext(&li)) != NULL) {
+        clusterManagerReshardTableItem *item = ln->value;
+        char *err;
+        if (opts & CLUSTER_MANAGER_CMD_FLAG_USE_ATOMIC_SLOT_MIGRATION) {
+            /* For atomic slot migration, we move everything as one command */
+            int result = clusterManagerMoveSlotRangesASM(item->source, target, item->slot_ranges, opts, &err);
+            if (!result) {
+                clusterManagerLogErr("clusterManagerMoveSlotRangeASM failed: %s\n", err);
```

Comment:
These error messages are to the user. They shouldn't normally contain references to the code internals.

How about something like this?

```suggestion
                clusterManagerLogErr("Atomic slot migration failed: %s\n", err);
```

Reply:
Makes sense